README.md

```diff
@@ -13,9 +13,6 @@ The protocol is described in detail in the [Morpho Blue Whitepaper](./morpho-blu
 
 ## Getting Started
```

Comment:
Is this section still needed ?

Reply:
Yes, it is a large title that contains smaller sections ^^